parser/parse.go

```diff
@@ -458,6 +458,7 @@ func (b *Builder) findTypesIn(pkgPath importPathString, u *types.Universe) error
 	for _, f := range b.parsed[pkgPath] {
 		if strings.HasSuffix(f.name, "/doc.go") {
 			tp := u.Package(string(pkgPath))
+			tp.Comments = []string{}
```

Comment:
The following three lines is a loop that append to tp.Comments. What do you mean by appending to "nil" slice?